### PR TITLE
Admin feature/see targets

### PR DESCRIPTION
**Pull request by MatiasCab**
#### Board:
* [RSLC-10 As an admin I should be able to see all the targets in the system](https://trello.com/c/q3ncuEKp)
---
#### Description:

---
#### Notes:
*
---
#### Tasks:
- [x] Add each element in this format
---
#### Risk:
*
---

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem 'annotate', '~> 3.2', '>= 3.0.3'
   gem 'bullet', '~> 7.0.3'
   gem 'byebug'
+  gem 'capybara'
```
Comment (rodrieiz) [inline Gemfile]: We should add version to prevent undesired updates

File: app/admin/targets.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    column :topic
+    column :created_at
+
+    actions
+  end
+
+  show do
+    attributes_table do
+      row :title
+      row 'Radius (m)', &:radius
```
Comment (rodrieiz) [inline app/admin/targets.rb]: maybe we can use i18n to override this column head

File: spec/features/targets/index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+describe 'Admin see targets', type: :feature do
+  let!(:admin) { create(:admin_user) }
```
Comment (rodrieiz) [inline spec/features/targets/index_spec.rb]: can we align this with the other hashes ?

File: spec/features/targets/show_spec.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  end
+
+  it 'show the target' do
+    subject
+    aggregate_failures do
+      expect(page).to have_content(first_target.title)
+      expect(page).to have_content(first_target.radius)
+      expect(page).to have_content(first_target.lat)
+      expect(page).to have_content(first_target.lng)
+      expect(page).to have_content(first_target.matched ? 'Yes' : 'No') # FIXME
```
Comment (rodrieiz) [inline spec/features/targets/show_spec.rb]: why this comment ?

**Review by rodrieiz — COMMENTED** [review]
💯

File: Gemfile (hunk 1 above)
Comment (MatiasCab) [reply Gemfile]: Ok!

File: app/admin/targets.rb (hunk 2 above)
Comment (MatiasCab) [reply app/admin/targets.rb]: Good idea!

File: spec/features/targets/index_spec.rb (hunk 3 above)
Comment (MatiasCab) [reply spec/features/targets/index_spec.rb]: Ok!

File: spec/features/targets/show_spec.rb (hunk 4 above)
Comment (MatiasCab) [reply spec/features/targets/show_spec.rb]: It is a mistake of my part

**Review by rodrieiz — APPROVED** [review]

**Review by enzofab91 — APPROVED** [review]
Nice work! 💯

**Pull request merged**